### ISSUE TITLE
Lens not showing AWS EKS clusters on MacOS Monterey - M1 chip

### ISSUE THREAD

**Issue by milosfolic** [kontena/lens#4405]
**Describe the bug**
Lens is not showing AWS EKS clusters on MacOS Monterey with Apple Silicon M1 chip.

**To Reproduce**
1. Install the latest Lens Apple Silicon version
2. Click on 'Catalog/Clusters'
3. Select kube config

**Expected behavior**
To see AWS EKS clusters as well.

**Screenshots**

**Environment (please complete the following information):**
- Lens Version: Lens: 5.2.7-latest.20211110.1
- OS: macOS Monterey version 12.0.1
- Installation method (e.g. snap or AppImage in Linux): .dmg file from [website](https://k8slens.dev/)

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
info:    ▪ 📟 Setting Lens as protocol client for lens:// +0ms
info:    ▪ 📟 Protocol client register succeeded ✅ +1ms
info:    ▪ 🚀 Starting Lens from "/Applications/Lens.app/Contents/MacOS/Lens" +37ms
info:    ▪ 🐚 Syncing shell environment +1ms
info:    ▪ 💾 Loading stores +326ms
info:    ▪ [LS]: LOADED from /Users/milosfolic/Library/Application Support/Lens/lens-user-store.json +10ms
warn:    ▪ [CLUSTER-STORE]: Failed to update/create a cluster: Error: ENOENT: no such file or directory, open '/Users/milosfolic/.kube/config-ogi' +14ms
warn:    ▪ [CLUSTER-STORE]: Failed to update/create a cluster: Error: ENOENT: no such file or directory, open '/Users/milosfolic/.kube/config-ogi' +0ms
warn:    ▪ [CLUSTER-STORE]: Failed to update/create a cluster: Error: ENOENT: no such file or directory, open '/Users/milosfolic/.kube/config-ogi' +0ms
info:    ▪ [AA]: LOADED from /Users/milosfolic/Library/Application Support/Lens/lens-cluster-store.json +1ms
info:    ▪ [IU]: LOADED from /Users/milosfolic/Library/Application Support/Lens/lens-hotbar-store.json +1ms
info:    ▪ [HT]: LOADED from /Users/milosfolic/Library/Application Support/Lens/lens-extensions.json +1ms
info:    ▪ [EC]: LOADED from /Users/milosfolic/Library/Application Support/Lens/lens-filesystem-provisioner-store.json +0ms
info:    ▪ [MA]: LOADED from /Users/milosfolic/Library/Application Support/Lens/lens-weblink-store.json +1ms
info:    ▪ 🔌 Starting LensProxy +4ms
info:    ▪ [LENS-PROXY]: Proxy server has started at 127.0.0.1:50707 +1ms
info:    ▪ 🔎 Testing LensProxy connection ... +1ms
info:    ▪ ⚡ LensProxy connection OK +64ms
info:    ▪ 🖥️  Starting WindowManager +2ms
info:    ▪ 🧩 Initializing extensions +43ms
info:    ▪ [EXTENSION-DISCOVERY] loading extensions from /Users/milosfolic/Library/Application Support/Lens +0ms
info:    ▪ [EXTENSION-INSTALLER] installing dependencies at /Users/milosfolic/Library/Application Support/Lens +34ms
info:    ▪ [WINDOW-MANAGER]: Loading Main window from url: http://localhost:50707 ... +66ms
info:    ▪ [EXTENSION-INSTALLER] dependencies installed at /Users/milosfolic/Library/Application Support/Lens +357ms
info:    ▪ [EXTENSION-DISCOVERY] watching extension add/remove in /Users/milosfolic/.k8slens/extensions +1ms
Native extraction module not loaded, msgpackr will still run, but with decreased performance. Cannot find module 'msgpackr-extract'
Lens Cloud Extension activated
info:    ▪ [C]: LOADED from /Users/milosfolic/Library/Application Support/Lens/extension-store/lenscloud-lens-extension/auth-token-store.json +28ms
info:    ▪ [C]: LOADED from /Users/milosfolic/Library/Application Support/Lens/extension-store/lens-survey/preferences-store.json +2ms
telemetry main extension activated
info:    ▪ [N]: LOADED from /Users/milosfolic/Library/Application Support/Lens/extension-store/lens-telemetry/preferences-store.json +20ms
info:    ┏ [IPC-RENDERER]: adding extension listener +42ms
info:    ┃ [1] {
info:    ┃ [2]   channel: 'deleteOneCluster',
info:    ┃ [3]   extension: { name: 'lenscloud-lens-extension', version: '5.2.7-latest.20211110.1' }
info:    ┗ [4] }
info:    ┏ [IPC-RENDERER]: adding extension listener +1ms
info:    ┃ [1] {
info:    ┃ [2]   channel: 'loadRemoteSpacesClusters',
info:    ┃ [3]   extension: { name: 'lenscloud-lens-extension', version: '5.2.7-latest.20211110.1' }
info:    ┗ [4] }
info:    ┏ [IPC-RENDERER]: adding extension listener +0ms
info:    ┃ [1] {
info:    ┃ [2]   channel: 'createOneSpace',
info:    ┃ [3]   extension: { name: 'lenscloud-lens-extension', version: '5.2.7-latest.20211110.1' }
info:    ┗ [4] }
info:    ┏ [IPC-RENDERER]: adding extension listener +0ms
info:    ┃ [1] {
info:    ┃ [2]   channel: 'leaveOneSpace',
info:    ┃ [3]   extension: { name: 'lenscloud-lens-extension', version: '5.2.7-latest.20211110.1' }
info:    ┗ [4] }
info:    ▪ [EXTENSION]: enabled lens-license@5.2.7-latest.20211110.1 +112ms
info:    ▪ [EXTENSION]: enabled lens-survey@5.2.7-latest.20211110.1 +2ms
info:    ▪ [EXTENSION]: enabled lens-telemetry@5.2.7-latest.20211110.1 +0ms
[CLUSTER-CONNECT] cluster proxy started on port 50720
[CLUSTER-CONNECT] cluster proxy is using aes-256-gcm for end-to-end encryption
info:    ▪ [EXTENSION]: enabled lenscloud-lens-extension@5.2.7-latest.20211110.1 +19ms
info:    ▪ [WINDOW-MANAGER]: Main window loaded +298ms
info:    ▪ [KUBECONFIG-SYNC]: starting requested syncs +282ms
info:    ┏ [KUBECONFIG-SYNC]: starting sync of file/folder +0ms
info:    ┗ [1] { filePath: '/Users/milosfolic/Library/Application Support/Lens/kubeconfigs' }
info:    ┏ [KUBECONFIG-SYNC]: starting sync of file/folder +0ms
info:    ┗ [1] { filePath: '/Users/milosfolic/.kube' }
info:    ┏ [KUBECONFIG-SYNC]: starting sync of file/folder +1ms
info:    ┗ [1] { filePath: '/Users/milosfolic/.kube/config' }
info:    ▪ 📡 Checking for app updates +0ms
info:    ▪ Checking for update +1ms
info:    ▪ [WINDOW-MANAGER]: Main window has reported that it has loaded +196ms
info:    ▪ [WINDOW-MANAGER]: Attaching webview +415ms
info:    ▪ Update for version 5.2.7-latest.20211110.1 is not available (latest version: 5.2.7-latest.20211110.1, downgrade is disallowed). +30ms
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
Same config works on Intel chip MacBook
```

**Additional context**

**Comment by ognjen-it** [comment]
+1

**Comment by Nokel81** [comment]
Is this resolved by the 5.3.0-beta?

**Comment by milosfolic** [comment]
@Nokel81 When I added one more cluster into the config everything is working as expected in 5.2.7-latest.20211110.1 version as well.
You can mark this as resolved!
